BDKSwiftExampleWallet/View/SettingsView.swift

```diff
@@ -15,13 +15,9 @@ struct SettingsView: View {
     @State private var isSeedPresented = false
 
     var body: some View {
-
-        ZStack {
-
-            Color(uiColor: UIColor.systemBackground)
-
-            VStack(spacing: 20.0) {
-
+        
+        Form {
+            Section(header: Text("Network Connection")) {
```

Comment:
How is "Client"?

Reply:
I think "Network" like it is in `main` is correct. Because what this is showing is which of the networks you are on:
 - bitcoin
 - testnet
 - signet
 - regtest
 
And then if the `getNetwork` function doesn't return anything (which would be a rare thing) we should probably just put "No Network" (I will add this part of the comment below again for line 35)